WooCommerce/UITestsFoundation/Screens/Orders/SingleOrderScreen.swift

```diff
@@ -38,20 +42,23 @@ public final class SingleOrderScreen: ScreenObject {
 
     @discardableResult
     public func verifySingleOrder(order: OrderData) throws -> Self {
+        let orderTotalPredicate = NSPredicate(format: "label CONTAINS %@", order.total)
+
         // Check that navigation bar contains order number
         let navigationBarTitles = app.navigationBars.map { $0.staticTexts.element.label }
         let expectedTitle = "#\(order.number)"
         XCTAssertTrue(navigationBarTitles.contains(where: { $0.contains(expectedTitle) }), "No navigation bar found with title \(expectedTitle)")
 
+        // Check order status and total
         let orderDetailTableView = app.tables["order-details-table-view"]
         orderDetailTableView.assertTextVisibilityCount(textToFind: order.status, expectedCount: 1)
-        orderDetailTableView.assertTextVisibilityCount(textToFind: order.total, expectedCount: 1)
+        XCTAssertTrue(app.otherElements.containing(orderTotalPredicate).element.exists)
 
-        // Expects 2 instances of first_name - one in Summary and one in Shipping details
-        orderDetailTableView.assertTextVisibilityCount(textToFind: order.billing.first_name, expectedCount: 2)
```

Comment:
this is not displayed on order 3337, i removed the check so that it works for all orders. i think should be fine even for the other to not have this check as we have already asserted other important details.